tools/rtpsrelay/GuidAddrSet.cpp

```diff
@@ -281,7 +281,7 @@ void GuidAddrSet::remove(const OpenDDS::DCPS::GUID_t& guid,
   relay_stats_reporter_.local_active_participants(guid_addr_set_map_.size(), now);
 
   if (config_.log_activity()) {
-    ACE_DEBUG((LM_INFO, "(%P|%t) INFO: GuidAddrSet::remove_i "
+    ACE_DEBUG((LM_INFO, "(%P|%t) INFO: GuidAddrSet::remove "
```

Comment:
```suggestion
    ACE_DEBUG((LM_INFO, "(%P|%t) INFO: GuidAddrSet::remove: "
```

Reply:
No other logging statement in this file uses a `:` here, so if we're going to change it we should make it consistent either by file or by project.

Reply:
Oh.  :(. Nevermind then.